src/Microsoft.Azure.WebJobs.Host/Bindings/Path/BindingTemplate.cs

```diff
@@ -36,6 +37,7 @@ internal BindingTemplate(string pattern, IReadOnlyList<BindingTemplateToken> tok
             _pattern = pattern;
             _tokens = tokens;
             _ignoreCase = ignoreCase;
+            _parameterResolver = parameterResolver ?? new DefaultParameterResolver();
```

Comment:
I think this is an important point. We need to be able to eventually have this on JobHostConfig. Are we confident there's a clear non-breaking path to do this? 

---
In reply to: [95934502](https://github.com/Azure/azure-webjobs-sdk/pull/971#discussion_r95934502) [](ancestors = 95934502)